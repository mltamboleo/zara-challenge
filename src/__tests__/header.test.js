import '@testing-library/jest-dom'
import { render } from '@testing-library/react'
import Header from '../app/components/header/index.tsx'

jest.mock('next/router', () => ({
  useRouter() {},
}));

describe('Header Component', () => {
  it('Renders Header', () => {
    const { container } = render(<Header />)
    expect(container).toMatchSnapshot()
  })
})