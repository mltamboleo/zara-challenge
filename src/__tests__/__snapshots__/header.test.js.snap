// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component Renders Header 1`] = `
<div>
  <div
    class="header"
  >
    <a
      class="header__logo-button"
      href="/"
    >
      <img
        alt="Home Page"
        class="header__logo"
        data-nimg="1"
        decoding="async"
        fetchpriority="high"
        height="52"
        src="/marvel-logo.svg"
        style="color: transparent;"
        width="130"
      />
    </a>
    <a
      class="header__fav-button"
      href="/"
      rel="nofollow"
    >
      <img
        alt="Home Page - Favorites"
        data-nimg="1"
        decoding="async"
        height="24"
        loading="lazy"
        src="/fav-default.svg"
        style="color: transparent;"
        width="24"
      />
      <span
        class="header__fav-number"
      >
        0
      </span>
    </a>
  </div>
</div>
`;
